### ISSUE TITLE
Roundcube 1.3.8: Cannot delete contacts from Roundcube Addressbook 

### ISSUE THREAD

**Issue by m0urs** [stwa/google_addressbook#52]
If I am trying to delete a contact from the Roundcube Collected Addressbook I get the error "Server Error! (Internal Server Error)" and in the error log of Roundcube I see

```
[14-Jan-2019 16:49:04 Europe/Berlin] PHP Fatal error:  Uncaught Error: Call to undefined function write_log() in /var/www/webmail.urspringer.de/html/plugins/google_addressbook/google_addressbook.php:208
Stack trace:
#0 /var/www/webmail.urspringer.de/html/program/lib/Roundcube/rcube_plugin_api.php(440): google_addressbook->contact_delete(Array)
#1 /var/www/webmail.urspringer.de/html/program/steps/addressbook/delete.inc(49): rcube_plugin_api->exec_hook('contact_delete', Array)
#2 /var/www/webmail.urspringer.de/html/index.php(303): include_once('/var/www/webmai...')
#3 {main}
  thrown in /var/www/webmail.urspringer.de/html/plugins/google_addressbook/google_addressbook.php on line 208

Any idea what I can do? Thanks a lot.

**Comment by Giu989** [comment]
I have this exact same issue on 1.3.9

**Comment by Giu989** [comment]
I had this exact same issue on 1.3.9

I fixed it by commenting out all the lines that called write_log() at the bottom of google_addressbook.php. Contacts can now be added and deleted without problem.

**Comment by vgropp** [comment]
this was fixed in #48 in master
